### PR TITLE
Quarkus 0.19.0 release post

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: _posts/2019-07-10-quarkus-0-19-released.adoc [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+We had a few issues with automatic schema upgrades handled by Hibernate ORM.
+
+Things are now smoother and we significantly improved the documentation to explain TODO link[how to configure schema management in various situations].
+
+=== New extensions
+
+==== MongoDB extension
+
+I don't think presenting https://www.mongodb.com/[MongoDB] is necessary anymore as it's probably the most well known NoSQL datastore.
```
Comment (geoand) [inline _posts/2019-07-10-quarkus-0-19-released.adoc]: Perhaps it would be better to say: `MongoDB needs no introduction....`

Comment (gsmet) [reply _posts/2019-07-10-quarkus-0-19-released.adoc]: Yup, done.

Comment (geoand) [reply _posts/2019-07-10-quarkus-0-19-released.adoc]: :+1:

File: _posts/2019-07-10-quarkus-0-19-released.adoc [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+== Contributors
+
+Quarkus has now https://github.com/quarkusio/quarkus/graphs/contributors[116 contributors].
+Many many thanks to each and everyone of them.
+
+In particular for this release, thanks to TODO list.
+
+== Come Join Us
+
+We value your feedback a lot so please report bugs, ask for improvements... Let's build something great together.
```
Comment (geoand) [inline _posts/2019-07-10-quarkus-0-19-released.adoc]: Perhaps replace the final period with an exclamation mark?

File: _posts/2019-07-10-quarkus-0-19-released.adoc [hunk 3]
```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: 'Quarkus 0.19.0 released'
+date: 2019-07-10
+tags: quarkus release
+author: gsmet
+---
+
+Quarkus 0.19.0 was just released and is now available on Maven Central.
```
Comment (emmanuelbernard) [inline _posts/2019-07-10-quarkus-0-19-released.adoc]: and is now available on Maven Central => isn't it a given? Or expected default?

File: _posts/2019-07-10-quarkus-0-19-released.adoc [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+====
+
+== What's new?
+
+Here are the main highlights of this release.
+
+=== Hibernate ORM schema management improvements
+
+We had a few issues with automatic schema upgrades handled by Hibernate ORM.
+
```
Comment (emmanuelbernard) [inline _posts/2019-07-10-quarkus-0-19-released.adoc]: I don't think you need a new paragraph between each sentence.

File: _posts/2019-07-10-quarkus-0-19-released.adoc [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+[NOTE]
+====
+Quarkus 0.19.0 is still targeting GraalVM 19.0.2 as we discovered a bug in GraalVM 19.1.0 that affects Quarkus.
+
+The issue is already fixed but we are waiting for a 19.1.1 release.
+====
+
+== What's new?
+
+Here are the main highlights of this release.
```
Comment (emmanuelbernard) [inline _posts/2019-07-10-quarkus-0-19-released.adoc]: Here are the main highlights of this release: MongoDB support, Neo4J support, AWS DynamoDB support,  Apache Tika support, Reactive Messaging MQTT support, better Hibernate ORM schema upgrade (useful for dev mode).

File: _posts/2019-07-10-quarkus-0-19-released.adoc [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+Neo4j offers Cypher, a declarative query language much like SQL.
+Cypher is used to query the graph, telling the database engine what patterns to match and how.
+Clients communicate over the so called Bolt protocol with the database.
+
+Quarkus now offers a Neo4j extension so that you can easily use Neo4j with Quarkus in JVM and native mode.
+
+A guide is coming soon but you can already take a look at https://github.com/quarkusio/quarkus/tree/master/integration-tests/neo4j[the integration tests] if you can't wait for it.
+
+==== Amazon DynamoDB extension
```
Comment (emmanuelbernard) [inline _posts/2019-07-10-quarkus-0-19-released.adoc]: I am curious why we let these not documented and not really configuration ready extensions in.

File: _posts/2019-07-10-quarkus-0-19-released.adoc [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+Future iterations will bring Quarkus-style config and the ability to inject the client directly with CDI.
+
+In the meantime, you can create your client manually and give us feedback about the configuration knobs you want us to expose.
+
+==== Apache Tika extension
+
+http://tika.apache.org/[Apache Tika] is a framework to extract data from documents (and especially binary ones such as PDF documents).
+
+With the Apache Tika Quarkus extension, just inject a `TikaParser` and you can extract the information from your documents.
+
```
Comment (emmanuelbernard) [inline _posts/2019-07-10-quarkus-0-19-released.adoc]: Again, I think you abuse paragraphs a lot in the blog entry.

File: _posts/2019-07-10-quarkus-0-19-released.adoc (hunk 6 above)
Comment (gsmet) [reply _posts/2019-07-10-quarkus-0-19-released.adoc]: I think it's an interesting contribution that can already be useful for early adopters (if you want to connect to Amazon DynamoDB in native mode).

We already have a follow-up PR to make progress in the right direction.

That's not something I would do post 1.0 but I think it's a good way to keep contributors with us and attract new ones, while we are still evolving quickly.

It's also related to a discussion we had about the platform coming and how to deal with externally contributed extensions.

Comment (gsmet) [reply _posts/2019-07-10-quarkus-0-19-released.adoc]: BTW, just to give a bit more context on these particular extensions, the authors are actively working on documentation/quarkification and so on.

They haven't dumped the extensions and run :).

See https://github.com/quarkusio/quarkus/pull/3171 or https://github.com/quarkusio/quarkus/pull/3172

File: _posts/2019-07-10-quarkus-0-19-released.adoc (hunk 7 above)
Comment (gsmet) [reply _posts/2019-07-10-quarkus-0-19-released.adoc]: Yeah, that was my impression too when reading the text in the UI. I improved the situation significantly.

**Pull request closed without merging**